manifests/w/WiresharkFoundation/Wireshark/4.4.0/WiresharkFoundation.Wireshark.installer.yaml

```diff
@@ -23,6 +28,12 @@ Installers:
   InstallerSha256: 6518C93481D2269D04158C7A632DDF912E6EB332B0E4DA4FD247D6E0E3D7D363
   ProductCode: '{6921416A-050F-4E48-B9DA-94F8941A09F0}'
   AppsAndFeaturesEntries:
-  - UpgradeCode: '{0D67AACE-269A-4264-81A3-DA8055C1C79C}'
+  - DisplayName: Wireshark
+    Publisher: The Wireshark developer community, https://www.wireshark.org/
+    DisplayVersion: 4.4.0.0

```

Comment:
```suggestion
```

> The `DisplayVersion` should be used carefully, as it can easily cause several issues. Here, it seems to only add a trailing `.0` to the `PackageVersion`, which the package manager does by default.
>
> _Originally posted at https://github.com/microsoft/winget-pkgs/pull/172687#issuecomment-2341904306_